core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/diskModels.scala

```diff
@@ -105,3 +106,6 @@ object FormattedBy extends Enum[FormattedBy] {
     override def asString: String = "GALAXY"
   }
 }
+
+final case class PvcId(asString: String) extends AnyVal
+final case class GalaxyRestore(galaxyPvcId: PvcId, cvmfsPvcId: PvcId, lastUsedBy: AppId)
```

Comment:
I moved away from having `disk` in the name because this data type represent all info needed for restoring a galaxy app, not just disk info.

Reply:
I'm happy to rename it if you have a suggestion here

Reply:
Some options:

`GalaxyPersistence`
`GalaxyRestoration`
`GalaxyRecovery`

Could also append to those above `Info` for further clarity (at the expense of verbosity).

Reply:
> I moved away from having disk in the name because this data type represent all info needed for restoring a galaxy app, not just disk info.

On a second thought - I'm not sure if that's accurate. We also need `chart` and `release` to restore a Galaxy app, which are not included in the class.

Reply:
hmmm..not sure I like any of the names...

> On a second thought - I'm not sure if that's accurate. We also need chart and release to restore a Galaxy app, which are not included in the class.

that's what `lastUsedApp` is for